balancer-js/src/modules/sdk.module.ts

```diff
@@ -32,13 +32,15 @@ export class BalancerSDK implements BalancerSDKRoot {
   balancerContracts: Contracts;
   zaps: Zaps;
   readonly networkConfig: BalancerNetworkConfig;
+  readonly provider: Provider;
```

Comment:
Is this being used somewhere?

Reply:
Good question, as far as I remember it was meant to replace public rpcProvider method. And it will be useful to migrate instantiation from SOR to root SDK. It's just a hint so far, not used anywhere yet.